src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a | b | c | d) && !(a & b & c & d) && !(a ^ b ^ c ^ d);
```

Comment:
Hint, you can start like this:
(a ^ b) && (c ^ d) || ....

Reply:
done